dcpy/lifecycle/package/generate_metadata_assets.py

```diff
@@ -12,12 +14,44 @@
 DEFAULT_DATA_DICTIONARY_STYLESHEET_PATH = RESOURCES_PATH / "data_dictionary.css"
 
 
+def _format_html(html: str) -> str:
+    return BeautifulSoup(html, "html.parser").prettify(formatter="html")
```

Comment:
I added and used this mainly for my sanity in the unit tests to get consistent outputs

this formatting doesn't seem to be how someone would naturally write html, but seems ok for now for templated/generated html